VD_Transmit/VD_Transmit.ino

```diff
@@ -0,0 +1,177 @@
+/* CAN-Bus transmit vehicle dynamics node*/
+
+#include <CAN.h>
+#include <millisDelay.h>
+#include "src/can_conv/sbm_network_definition.h"
+
+#define SERIAL_DEBUG_PRINTS_ON 1
+
+/*******************************************************************************/
+
+#include <Adafruit_LSM6DSOX.h>
+Adafruit_LSM6DSOX lsm6ds;
+
+#include <Adafruit_LIS3MDL.h>
+Adafruit_LIS3MDL lis3mdl;
+
+/*******************************************************************************/
```

Comment:
You don't need to decorate it like i did with the example code. I trying to be overly illustrative but your code is fairly straightforward. 

Also, typically you want to have all your includes up top. Feel free to disregard the suggestion. just showing off the github features. 

```suggestion

#include <Adafruit_LSM6DSOX.h>
#include <Adafruit_LIS3MDL.h>
#define SERIAL_DEBUG_PRINTS_ON 1

Adafruit_LSM6DSOX lsm6ds;
Adafruit_LIS3MDL lis3mdl;


```

Reply:
I like to decorate my code to make it easier to find things. Sometimes my brain doesn't work at full capacity, so the less thinking I need to do in the future, the better.

I can move the includes to the top. That snippet was copied over from Adafruit's example code, and I never bothered to re-arrange it.